emcc.py

```diff
@@ -99,7 +99,7 @@ class Intermediate(object):
   counter = 0
 def save_intermediate(name=None, suffix='js'):
   name = os.path.join(shared.get_emscripten_temp_dir(), 'emcc-%d%s.%s' % (Intermediate.counter, '' if name is None else '-' + name, suffix))
-  if type(final) != str:
+  if isinstance(final, list):
```

Comment:
could this check the type is `list`? or is this the better notation?

Reply:
Both work, the difference is that `isinstance` checks class inheritance. I used it just to suppress Pylint warnings, though.

Reply:
Ok, cool.